views/js/qtiCreator/test/MathEditor/test.js

```diff
@@ -0,0 +1,67 @@
+define([
+    'jquery',
+    'taoQtiItem/qtiCreator/editor/MathEditor'
+],
+function($, MathEditor) {
+    'use strict';
+
+    QUnit.asyncTest('Latex rendering', function test(assert) {
+        var texExprs = [
```

Comment:
We have a plugin to achieve parameterized testing, you should use it. Take the following test as an example : https://github.com/oat-sa/tao-core/blob/develop/views/js/test/core/encoder/str2array/test.js